MsCorePkg/Library/PlatformBootManagerLib/BdsPlatform.c

```diff
@@ -41,11 +41,15 @@ ExitPmAuth (
 
   DEBUG ((DEBUG_INFO, "ExitPmAuth ()- Start\n"));
 
+  BeforeEndOfDxeEventHook (); // MS_CHANGE

```

Comment:
Since this is in MU_PLUS, we should not need to use `MS_CHANGE` tags in this PR.